invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -436,3 +438,8 @@ dl.details-list {
 .shadowless {
   box-shadow: none !important;
 }
+
+.error-colored {
```

Comment:
There are props in semantic ui to implement error highlighting, is there any reason for not using the theme provided classes ?

Reply:
it is because we wanted a lighter error message, because red is very bright. So i added this custom class to apply the lighter red found for example in a form label prompt

Reply:
could you include a screenshot please?

Reply:
you can see it here https://github.com/inveniosoftware/invenio-communities/pull/465

Reply:
I think it should be consistent with the rest of the error messages which are using the bright red, please remove the custom CSS

Reply:
then this PR can be closed